### PR TITLE
Site Management Dashboard: Use smaller margins on smaller screens

### PR DESCRIPTION
**Pull request by zaguiini**
#### Proposed Changes

Let's conform to Calypso and give the elements more room to breathe by using half the current margins on smaller viewports. Decreased from 32px to 16px, both vertically and horizontally.

| Before | After |
| ------ | ------ |
| old | new |

### CODE REVIEW

File: client/sites-dashboard/components/sites-content-controls.tsx [hunk 1]
```diff
@@ -26,6 +26,10 @@ const FilterBar = styled.div( {
 	'@media screen and (min-width: 660px)': {
 		flexDirection: 'row',
 	},
+
+	'@media screen and (max-width: 781px)': {
```
Comment (zaguiini) [inline client/sites-dashboard/components/sites-content-controls.tsx]: I will create helpers so we can standardize the breakpoints definition.

Comment (danielbachhuber) [reply client/sites-dashboard/components/sites-content-controls.tsx]: > I will create helpers so we can standardize the breakpoints definition.

@zaguiini In this pull request or another?

Comment (zaguiini) [reply client/sites-dashboard/components/sites-content-controls.tsx]: This one. Just pushed a commit 👍

**Review by danielbachhuber — APPROVED** [review]
❤️ Love it!

File: client/sites-dashboard/components/sites-content-controls.tsx (hunk 1 above)
Comment (p-jackson) [reply client/sites-dashboard/components/sites-content-controls.tsx]: These standard breakpoints are already defined in Gutenberg, but they're only in `SCSS`! :shakes-fist:

https://github.com/WordPress/gutenberg/blob/8b9db3dc025f73e9dad9b076c6f14394b898a9b5/packages/base-styles/_breakpoints.scss#L9

**Pull request merged**